src/QueryHandler/QueryHandler.php

```diff
@@ -81,7 +74,7 @@ public function __construct(array $contextProviders = [])
      * @param array|null $vars
      * @return array
      */
-    public function query(GraphQLSchema $schema, $query, ?array $vars = []): array
+    public function query(GraphQLSchema $schema, string|DocumentNode $query, ?array $vars = []): array
```

Comment:
Not supported in PHP 7.4
```suggestion
    public function query(GraphQLSchema $schema, $query, ?array $vars = []): array
```